src/psqlgraph/hydrator.py

```diff
@@ -4,7 +4,8 @@
 import random
 import uuid
 from collections import defaultdict, deque
-from typing import Dict, Iterable, List, Optional, Union
+from collections.abc import Iterable
+from typing import Dict, List, Optional, Union
```

Comment:
Can you drop Dict and List? I looks like you are using the native `dict` and `list` below.